frontend/src/components/restaurants/RestaurantCardItem.tsx

```diff
@@ -3,26 +3,35 @@ import DeleteIcon from "@mui/icons-material/Delete";
 import EditIcon from "@mui/icons-material/Edit";
 import React from "react";
 import {RestaurantCard} from "../../models/restaurants/RestaurantCard";
-import {Box, Rating, styled} from "@mui/material";
+import {Box, Rating} from "@mui/material";
 import EuroIcon from "@mui/icons-material/Euro";
-import image from "../result/dummypic.jpg";
+import image0 from "./burger.jpg"
+import image1 from "./japaneseRestaurant.jpg"
+import image2 from "./pizza.jpg"
+import { StyledRating } from "../../models/restaurants/StylingRating";
 
 interface RestaurantCardItemProps {
     restaurantCard: RestaurantCard
     deleteRestaurantCard: Function,
     editRestaurantCard: Function,
+    index: number
 }
 
-export default function RestaurantCardItem({restaurantCard, deleteRestaurantCard, editRestaurantCard}:RestaurantCardItemProps) {
+export default function RestaurantCardItem({index, restaurantCard, deleteRestaurantCard, editRestaurantCard}:RestaurantCardItemProps) {
+    if(index>2) {
+        index=2;
+    }
 
-    const StyledRating = styled(Rating)({
-        '& .MuiRating-iconFilled': {
-            color: 'hsl(222, 100%, 50%)',
-        },
-        '& .MuiRating-iconHover': {
-            color: 'hsl(222, 100%, 41%)',
-        },
-    });
+    let selectedImage;
+    if (index === 0) {
+        selectedImage = image0
+    }
+    if (index === 1) {
+        selectedImage = image1
+    }
+    if (index === 2) {
+        selectedImage = image2
+    }
```

Comment:
We already talked about this. 

But maybe choose one of your images as default?